FAQ.md

```diff
@@ -758,6 +860,18 @@ Yes. TiDB uses the cost-based optimizer. The cost model and statistics are const
 
 View the `Healthy` field using `show stats_healthy` and generally you need to execute `analyze` on a table when the field value is smaller than 60.
 
+#### What is the ID rule when a query plan is presented as a tree? What is the execution order for this tree?
+
+No rule exists for these IDs but the IDs are unique. When IDs are generated, a counter works and ID will plus one when one plan is generated. The execution order has nothing to do with the ID. The whole query plan is a tree and the execution process starts from the root node and the data is returned to the upper level continuously. For details about the query plan, see [Understanding the TiDB Query Execution Plan](sql/understanding-the-query-execution-plan.md).
+
+#### In the TiDB query plan, `cop` tasks are in the same root. Are they executed concurrently?
+
+Currently the computing tasks of TiDB to two different tasks: `cop task` and `root task`. 
+
+`cop task` is the computing task which is pushed down to the KV end for execution distributedly; `root task` is the computing task executed for single point of execution on TiDB end. 
```

Comment:
-> `cop task` is the computing task which is pushed down to the KV end for distributed execution; `root task` is the computing task for single point execution on the TiDB end.